src/gba/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(SOURCE_FILES
 	cheats/parv3.c
 	core.c
 	dma.c
+	extra/extensions.c
```

Comment:
Extras go in the `EXTRA_FILES` section below, but the feature will need to be able to be ifdef'd out for that to work.

Reply:
Should I ifdef'd under MINIMAL_CORE or do I create a new define?